### ISSUE TITLE
Chrome and PoE Trade Notifications

### ISSUE THREAD

**Issue by TKluwe** [exile-center/better-trading#134]
I can't get the numbers to show on livesearches when I enable this extension using chrome.  When multiple searches are open this is useful, because I can see which one dinged by looking for the number.  This issue is recent as of today.  If I remove the extension and relaunch chrome it works.

**Comment by GhoulSlayer001** [comment]
Yes, the same problem appeared yesterday, I thought it was only me

**Comment by dbjorge** [comment]
Thanks for reporting the issue, I'll take a look tomorrow

**Comment by dbjorge** [comment]
I investigated what's going on here. For background, the trade site itself actually uses 2 different methods for adding a number to trade site tabs:

1) It updates the page's favicon image with a red number badge, like this: 
![favicon badge example](https://github.com/exile-center/better-trading/assets/376284/7d82cfaa-4d54-4fbf-9f6e-e6b0003a73a2)
2) It updates the page title text to add the number in parentheses at the front of the title (like `(3) Trade - Path of Exile`)

The Better Trading extension includes a feature that updates trade page titles based on the search you're running, so page titles look like `Watcher's Eye Prismatic Jewel - Trade - Path of Exile` instead of just `Trade - Path of Exile`. This functionality overrides method 2 above, but previously this wasn't a big deal since it was redundant with method 1 anyway.

However, recently, it looks like a new issue in the trade site itself (not this extension) has broken notification method 1 above. Technical details (which I've already reported separately to GGG): to generate the favicon-with-badge image, it needs to make a CORS-enabled request to fetch the base `favicon.ico` file such that it can be modified in a hidden canvas element - until very recently, `poecdn.com` enabled this by including an `Access-Control-Allow-Origin: *` header in responses for `favicon.ico`, but recently it's begun omitting that header.

As a workaround, I'll look into updating how the page-retitling feature works to get it to preserve the number in parentheses at the start of the title.
